jep-012-raw-string-literals.md

```diff
@@ -6,6 +6,7 @@
 | **Author** | Michael Downling
 | **Status** | accepted
 | **Created**| 09-Apr-2015
+| **Obsoleted By**| [JEP-12-01](./jep-012-01-raw-string-literals.md)
```

Comment:
This PR should update the README to document JEP naming conventions. I don't think the JEP-originalNumber-updateNumber pattern is bad, but it is non-obvious.